package-lock.json

```diff
@@ -1,21 +1,23 @@
 {
-  "name": "atlas-blueprint-blog-starter",
+  "name": "atlas-blueprint-portfolio",
   "version": "0.1.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "name": "atlas-blueprint-blog-starter",
+      "name": "atlas-blueprint-portfolio",
       "version": "0.1.0",
       "dependencies": {
         "@faustjs/core": "^0.15.2",
         "@faustjs/next": "^0.15.3",
+        "gqty": "^2.2.0",
```

Comment:
Does this import affect the bundle size? I was wondering to see because we already use GQty in `Faust.js`

Reply:
It should not, not by much. `gqty` will already be in the user's `node_modules` dir since Faust.js relies on it. This is just explicitly including it.